packages/function-once/index.js

```diff
@@ -0,0 +1,25 @@
+module.exports = once;
+
+/*
+let i = 0;
+const getFirst = once(() => ++i);
+getFirst(); // 1
+getFirst(); // 1
+*/
+
+function once(fn) {
+  var called, value;
+
+  if (typeof fn !== 'function') {
+    new Error('expected a function but got ' + fn);
```

Comment:
Oh, I just assumed that the tests folder was required by reading the file system.
Will update accordingly. 😛